src/components/inputs/RadioButton/RadioButton.scss

```diff
@@ -0,0 +1,81 @@
+$C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-radio-button;
+
+.#{$C} {
+  position: relative;
+  display: flex;
+
+  input[type=radio] {
+    all:unset;
+  }
+
+  input[type=radio]:before {
+    content: '';
+    box-sizing: border-box;
+    border-radius: 50%;
+    display: inline-block;
+    width: 16px;
```

Comment:
I would prefer relative dimensions (percentages), so one can easily scale the radio button.